### PR TITLE
Add supreme pizza

### PR DESCRIPTION
**Pull request by HipAndroid**
#### Summary
```SUMMARY: Content "Adds a recipe for a new pizza that uses a variety of ingredients."```

#### Purpose of change

Rewards survivors who've scavenged, farmed, hunted, and foraged a wide variety of ingredients with a relatively resource intense source of joy and nutrition.

#### Describe the solution

Adds "Supreme Pizza" comestible and a new recipe found in Cucina Italiana.

#### Describe alternatives you've considered

Limiting toppings to more traditional toppings such as sausage and mushrooms.

#### Testing

Spawned pizza, ate it. Spawned Cucina Italiana and necessary components to craft the pizza, crafted it, ate it. 

#### Additional context

This is my first PR! Started small to get a better grip on the system.

### CODE REVIEW

File: data/json/items/comestibles/meat_dishes.json [hunk 1]
```diff
@@ -775,6 +775,21 @@
...
+    "name": "supreme pizza",
+    "copy-from": "pizza_meat",
+    "looks-like": "pizza_meat",
+    "color": "light_red",
+    "weight": "300 g",
+    "calories": 641,
+    "description": "A supreme pizza with ALL the toppings.",
+    "price": 1500,
+    "material": [ "wheat", "flesh", "veggy", "milk" ],
+    "fun": 20,
```
Comment (I-am-Erk) [inline data/json/items/comestibles/meat_dishes.json]: This fun score is pretty out of line, pizza_meat is already an extremely high fun score. I'd suggest a Supreme pizza might be 11. Adding some peppers and toppings is great, but it doesn't turn an already extremely tasty food into something on a par with a shot of heroin.

File: data/json/recipes/recipe_food.json [hunk 2]
```diff
@@ -4142,6 +4142,46 @@
...
+        [ "rehydrated_fish", 2 ],
+        [ "dry_fish", 2 ]
+      ],
+      [
+        [ "veggy_any", 2, "LIST" ],
+        [ "mushroom", 2 ],
+        [ "dry_mushroom", 2 ],
+        [ "morel_cooked", 2 ],
+        [ "tofu", 2 ],
+        [ "dry_tofu", 2 ],
```
Comment (I-am-Erk) [inline data/json/recipes/recipe_food.json]: Perhaps the tofus would be better put on the meat list? In terms of recipe, they're more appropriate there.

File: data/json/recipes/recipe_food.json [hunk 3]
```diff
@@ -4142,6 +4142,46 @@
...
+        [ "mushroom_cooked", 2 ],
+        [ "dry_veggy", 2 ]
+      ],
+      [ [ "cheese", 2 ], [ "can_cheese", 2 ], [ "cheese_hard", 2 ] ],
```
Comment (I-am-Erk) [inline data/json/recipes/recipe_food.json]: canned cheese on a pizza? get out of my kitchen and never come back.

(just kidding it's fine but don't do this in real life or your friends will all abandon you)

File: data/json/items/comestibles/meat_dishes.json (hunk 1 above)
Comment (HipAndroid) [reply data/json/items/comestibles/meat_dishes.json]: Fair enough! I'd taken the fun score from other high-effort foods like the Milkshake but was still a little unsure on where exactly to place it.

**Comment by ThomasLinkin** [comment]
almost there. keep it up my bro. welcome to the contribution

**Pull request merged**